crates/edr_runtime/src/transaction.rs

```diff
@@ -34,11 +34,13 @@ pub fn validate<TransactionT: Transaction>(
     transaction: TransactionT,
     spec_id: EvmSpecId,
 ) -> Result<TransactionT, CreationError> {
+    const EIP7623_DISABLED: bool = false;
+
     if transaction.kind() == TxKind::Create && transaction.input().is_empty() {
         return Err(CreationError::ContractMissingData);
     }
 
-    match validate_initial_tx_gas(&transaction, spec_id) {
+    match validate_initial_tx_gas(&transaction, spec_id, EIP7623_DISABLED) {
```

Comment:
REMV introduced the ability to disable EIP-7623. For now, EDR doesn't plan to support disabling this cap.

See discussion at: https://www.notion.so/Fusaka-26f578cdeaf58031a045cd51399bea74?d=2a7578cdeaf580bfa1d0001cdd6af8e1&source=copy_link#2a4578cdeaf58023b5f6e1e6c88a1d8f